libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/arrow/RouteArrowUtils.kt

```diff
@@ -262,4 +260,13 @@ internal object RouteArrowUtils {
             style.styleLayerExists(RouteLayerConstants.ARROW_SHAFT_LINE_LAYER_ID) &&
             style.styleLayerExists(RouteLayerConstants.ARROW_HEAD_LAYER_ID)
     }
+
+    internal fun removeLayers(style: Style) {
```

Comment:
Just removing the layers creates kind of strange state because the layer sources and the images used for the arrow heads are still in the map style. Even thought the `initialize` method in `RouteArrowUtils` currently checks for those items there's no guarantee the implementation won't change and if it did an error condition would be created that isn't caught by any tests.  I suggest creating a reset method that removes the styles, layers and image resources.

Reply:
Thanks, I updated the function to also remove the images. Do you think I should add a public function to `MapboxRouteArrowView` that calls `RouteArrowUtils.removeLayers`?

Reply:
I"m not sure.  It might be best to wait and think through the consequences. At the moment I can't think of any unintended side effects but I'm not 100% confident.  If the method isn't currently needed I suggest waiting.